tmt/steps/execute/__init__.py

```diff
@@ -581,8 +581,17 @@ def wake(self) -> None:
 
     def summary(self) -> None:
         """ Give a concise summary of the execution """
-        tests = fmf.utils.listed(self.results(), 'test')
-        self.info('summary', f'{tests} executed', 'green', shift=1)
+        executed_tests = filter(lambda r: r.result != ResultOutcome.SKIP, self.results())
```

Comment:
Could we use list comprehensions as in the rest of the code? Thus
```suggestion
        executed_tests = [r for r in self.results() if r.result != ResultOutcome.SKIP ]
```

Similar bellow for 'skipped_tests'

Reply:
Should be covered by 2f98658f.